src/utils/gemini.js

```diff
@@ -0,0 +1,44 @@
+import { API_KEY, MODEL } from '../env_config.js';
+
+// Define the API endpoint for the Gemini model, using an environment variable for the API key.
+const Gemini_Endpoint = `https://generativelanguage.googleapis.com/v1beta/models/${MODEL}:generateContent?key=${API_KEY}`;
+
+const postGeminiPrompt = async (prompt, tabs) => {
+  // Construct the request body
+  const postData = {
+    contents: [
+      {
+        parts: [
+          {
+            text: `Given a list of tab title ${tabs}. Pick titles and return their exact title without explaination that are relate to the prompt "${prompt}". If there is no match return "There is no match"`,
```

Comment:
Fine prompt. Anh suggest nên 
- Given URL nữa vì nhiều tab không có (hoặc trùng) title
- Ghi chi tiết trong prompt return datatype mà em expect luôn. Ex: "a list of tabs' titles separated by commas. If there is no match, return an empty list"